packages/appstore/src/components/appstore-banner/appstore-banner.tsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import SVGMigrationBanner from './svg-migration-banner';
+import { useMT5SVGEligibleToMigrate } from '@deriv/hooks';
 
 const AppstoreBanner = () => {
-    //TODO: will add proper condition check when API is ready
-    if (true) return <SVGMigrationBanner />;
+    const { has_svg_accounts_to_migrate } = useMT5SVGEligibleToMigrate();
+
+    return has_svg_accounts_to_migrate ? <SVGMigrationBanner /> : null;
```

Comment:
does `has_svg_accounts_to_migrate && <SVGMigrationBanner />` work here?

Reply:
We can't do that because it is expected to return a value in `appstore` have to include `null` inside.